packages/core/src/components.d.ts

```diff
@@ -180,7 +180,7 @@ export namespace Components {
     }
     interface IxChip {
         /**
-          * Display chip in active state. Only works with `variant="primary"`
+          * Determinate if the chip is interactive. If active is false no closable icon is visible and no interaction is possible.
```

Comment:
Determines if the chip is interactive. If false no user input (e.g. mouse states, keyboard navigation) will be possible and also the close button will not be present.